Zend/zend_attributes.c

```diff
@@ -315,6 +384,17 @@ void zend_register_attribute_ce(void)
 	zend_ce_allow_dynamic_properties = register_class_AllowDynamicProperties();
 	attr = zend_internal_attribute_register(zend_ce_allow_dynamic_properties, ZEND_ATTRIBUTE_TARGET_CLASS);
 	attr->validator = validate_allow_dynamic_properties;
+
+	zend_ce_sensitive_parameter = register_class_SensitiveParameter();
+	attr = zend_internal_attribute_register(zend_ce_sensitive_parameter, ZEND_ATTRIBUTE_TARGET_PARAMETER);
+
+	/* This is not an attribute, thus the register call is missing. */
+	zend_ce_sensitive_parameter_value = register_class_SensitiveParameterValue();
+	zend_ce_sensitive_parameter_value->create_object = attributes_sensitive_parameter_value_new;
+	memcpy(&attributes_object_handlers_sensitive_parameter_value, &std_object_handlers, sizeof(zend_object_handlers));
+	attributes_object_handlers_sensitive_parameter_value.offset = XtOffsetOf(attributes_sensitive_parameter_value, std);
```

Comment:
nit: This is always 0 and doesn't need to be overridden with the current design.